libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -549,18 +549,11 @@ - (void)setupServiceConfiguration
 - (void)logoutCleanup
 {
     self.passcodeDisplayed = NO;
-    self.screenLockDisplayed = NO;
 }
 
 - (void)handleAppForeground:(NSNotification *)notification
 {
-    [SFSDKCoreLogger d:[self class] format:@"App is entering the foreground."];
-    if (self.isScreenLockDisplayed) {
-        // Screen Lock was already displayed prior to app foreground.
-        [SFSDKCoreLogger i:[self class] format:@"%@ Screen Lock already displayed.", NSStringFromSelector(_cmd)];
-    } else {
-        [[SFScreenLockManager shared] handleAppForground];
-    }
+    [[SFScreenLockManager shared] handleAppForground];
```

Comment:
@brandonpage Sorry for the late comment, but there's a typo on `handleAppForeground`. Could you fix it?